spec/test_helpers/test_helpers.rb

```diff
@@ -0,0 +1,41 @@
+module TestHelpers
+
+  def register(user)
+    visit '/'
+    click_link 'Register'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    fill_in 'Password confirmation', with: user.password_confirmation
+    click_button 'Register'
+  end
+
+  def log_in(user)
+    visit '/'
+    click_link 'Log in'
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+  end
+
+  def add_picture
+    click_link 'New Post'
+    attach_file('picture[image]', File.join(Rails.root,'spec',"files", 'images', 'duck.jpg'))
```

Comment:
Metrics/LineLength: Line is too long. [93/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)